### PR TITLE
Update the main program as soon as you see this.

### PR DESCRIPTION
**Pull request by Neal-Cyber2020**
When will I have write access!?

### CODE REVIEW

**Comment by icttechinc** [comment]
idk man i tried googling but there's no actual method... not that i can find...

**Comment by Neal-Cyber2020** [comment]
I searched it up, and found it immediately, don't know what you've been
doing. I searched up "how do you give someone write access in github" and
the first 2 links show it. Here's what's in the first link:
Search Results
Featured snippet from the web
Under your repository name, click Settings. In the left sidebar, click
Collaborators & teams. Under "Collaborators", type the name of the person
you'd like to *give access* to the repository, then click Add collaborator.
Next to the new collaborator's name, choose the appropriate *permission*
 level: *Write*, Read, or Admin

>

File: main.py [hunk 1]
```diff
@@ -28,22 +28,85 @@
...
+        def __init__(self,nb,c,m,vm,ic):
+    	   	self.New_Board = nb
+            self.Check = c
+            self.Mate = m
+            self.Valid_Move = vm
+            self.Is_Checked = ic
+    class Pieces:                                                                       #Pieces General Code
+        def __init__ (self,c,t,it,ia):
+            self.captured = c
+            self.take = t
```
Comment (Neal-Cyber2020) [inline main.py:41]: testing

**Comment by Neal-Cyber2020** [comment]
Accept the pull request! Why didn't you accept the pull request?

**Pull request merged**